src/main/resources/db/migration/V212__remove_tec_court_for_money_claims.sql

```diff
@@ -0,0 +1,10 @@
+--FACT-2524
+--Remove the entry in search_serviceareacourt that points to traffic enforcement centre tec for money claims
+--CNBC will remain and system will point there instead
+
+WITH tec_id AS (
+  SELECT id FROM search_court WHERE slug IN ('traffic-enforcement-centre-tec') LIMIT 1
+)
+DELETE FROM search_serviceareacourt ssac
+WHERE ssac.court_id = (SELECT id FROM tec_id)
+AND ssac.catchment_type = 'national';
```

Comment:
Would be good if we included the servicearea_id as part of this to ensure we only delete it from money claims even if it doesn't exist for any other service area it is still safer. You can check the prod DB to confirm the service area ID etc or join it from a different table.